vmdb/app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -9,7 +9,7 @@ def textual_group_properties
   end
 
   def textual_group_relationships
-    items = %w(ems)
+    items = %w(ems container_groups container_services container_routes container_replicators)
```

Comment:
Order should be:

    ems container_routes container_services container_replicators container_groups